src/ray/test/run_object_manager_tests.sh

```diff
@@ -30,12 +30,11 @@ echo "$REDIS_DIR/redis-server --loglevel warning --loadmodule $REDIS_MODULE --po
 echo "$REDIS_DIR/redis-cli -p 6379 shutdown"
 
 # Allow cleanup commands to fail.
-killall plasma_store || true
-$REDIS_DIR/redis-cli -p 6379 shutdown || true
+# killall plasma_store || true
+# $REDIS_DIR/redis-cli -p 6379 shutdown || true
```

Comment:
Why are we commenting these out?

Reply:
They'll be removed or uncommented. The tests now start and remove plasma stores by `pid`.